### PR TITLE
Service catalog UI update

### PR DESCRIPTION
**Pull request by Krishanx92**

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx [hunk 1]
```diff
@@ -64,6 +66,10 @@ const styles = (theme) => ({
     buttonIcon: {
         marginRight: 10,
     },
+    buttonWarningColor: {
+        marginRight: 10,
+        color: '#ffcc00',
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx:71]: Shall we use a color from theme ?

Example: [Codesandbox](https://codesandbox.io/s/material-demo-forked-4sgvo?file=/demo.js)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx [hunk 2]
```diff
@@ -458,6 +498,76 @@ class APIDefinition extends React.Component {
                         {api.type !== 'APIProduct' && (
                             <ImportDefinition setSchemaDefinition={this.setSchemaDefinition} />
                         )}
+                        {api.serviceInfo && api.serviceInfo.outdated && (
+                            <div>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx:502]: Shall we move this to a new component ?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx (hunk 1 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx (hunk 2 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: Fixed

**Pull request merged**